init-db.py

```diff
@@ -0,0 +1,20 @@
+from models import database, Item, User, Order, OrderItem
+
+database.connect()
+
+
+# Inizializzo il db cancellando tutte le tabelle
+Item.drop_table(fail_silently=True)
+User.drop_table(fail_silently=True)
+Order.drop_table(fail_silently=True)
+OrderItem.drop_table(fail_silently=True)
+
+
+# Ricreo le tabelle nel db
+Item.create_table(fail_silently=True)
+User.create_table(fail_silently=True)
+Order.create_table(fail_silently=True)
+OrderItem.create_table(fail_silently=True)
+
+
+database.close()
```

Comment:
il codice eseguibile deve essere sempre all'interno di una funzione, eseguito solo se:
```
if __name__ == '__main__':
```
in questo modo puoi andare ad importare lo script come modulo ed usare le funzioni definite. Creiamo quindi una funzione per eliminare le tabelle ed una per crearle. 
PS: ma se viene fatta la `drop_table` e poi la `create_table`, perché quest'ultima ha `fail_silently=True`?